libs/sdk-ui-filters/src/DateFilter/DateFilterButton/DateFilterButton.tsx

```diff
@@ -53,8 +45,10 @@ export const DateFilterButton: React.FC<IDateFilterButtonProps> = ({
                 isOpen && "is-active",
                 disabled && "disabled",
             )}
-            onKeyDown={onKeyDown}
             role="button"
+            aria-haspopup="dialog"
+            aria-expanded={isOpen}
+            aria-controls={isOpen ? dropdownId : undefined}
             tabIndex={0}
```

Comment:
do we need keep it focusable if there is no handler now?

Reply:
Discussed, irrelevant.